src/components/PasswordInput.component.tsx

```diff
@@ -0,0 +1,23 @@
+import React, { FC, FormEvent } from "react";
+
+interface PasswordInputBoxProps {
+  placeholder: string;
+  handleChange: (e: FormEvent) => void;
+}
+
+const PasswordInput: FC<PasswordInputBoxProps> = ({
+  placeholder,
+  handleChange,
+}) => {
```

Comment:
If your component is only returning some JSX/TSX as in this case, you can remove the curly braces and the 'return' keyword. It'll look like this: 

`const PasswordInput: FC<PasswordInputBoxProps> = ({
  placeholder,
  handleChange,
}) => (
    <input
      ...
    />
  );`